### DISCUSSION TITLE
DatePicker not only a single day, but also multiple days

### DISCUSSION THREAD

**Discussion by gual11** [flet-dev/flet#4597] [Ideas]
Would be helpful, that the DatePicker is also able to select multiple consecutive days. Like when you go on booking.com and select the days for a holiday

**Comment by ndonkoHenri** [comment]
Perhaps in the future we could look into implementing packages like [date_picker_plus](https://pub.dev/packages/date_picker_plus).

**Reply by gual11** [reply]
yes I think that would be helpful, exactly what i meant
